src/receivers/ExpressReceiver.ts

```diff
@@ -217,16 +220,21 @@ export default class ExpressReceiver implements Receiver {
   }
 
   // TODO: can this method be defined as generic instead of using overloads?
-  public start(port: number): Promise<Server>;
-  public start(portOrListenOptions: number | ListenOptions, serverOptions?: ServerOptions): Promise<Server>;
+  // TODO: if a router is provided, this is no-op, and so the parameters are useless. How to resolve this?
+  public start(port: number): Promise<Server | void>;
+  public start(portOrListenOptions: number | ListenOptions, serverOptions?: ServerOptions): Promise<Server | void>;
   public start(
     portOrListenOptions: number | ListenOptions,
     httpsServerOptions?: HTTPSServerOptions,
-  ): Promise<HTTPSServer>;
+  ): Promise<HTTPSServer | void>;
   public start(
     portOrListenOptions: number | ListenOptions,
     serverOptions: ServerOptions | HTTPSServerOptions = {},
-  ): Promise<Server | HTTPSServer> {
+  ): Promise<Server | HTTPSServer | void> {
+    if (!this.app) {
+      return Promise.resolve();
```

Comment:
How about adding logs here? For instance,

```js
this.logger.info('As the Express app instance is not managed by this receiver, #start() method does nothing.');
```

or something like that. If it's fine, we can do the same for the stop() method

Reply:
Ya that seems like a good idea to me!